dev-util/bencode/bencode_tools-20110315.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="C and Python tools for manipulating bencoded data"
+DESCRIPTION="bencode-tools is a collection of tools for manipulating \
+bencoded data. See http://en.wikipedia.org/wiki/Bencode for information \
+on bencode format.
+
+This was formerly named typevalidator. With the introduction of \
+libbencodetools the project is named bencode-tools."
+HOMEPAGE="http://zakalwe.fi/~shd/foss/bencode-tools/"
+COPYRIGHT="2002-2005 Bram Cohen and Ross Cohen"
+LICENSE="Bencode"
+REVISION="1"
+SOURCE_URI="http://zakalwe.fi/~shd/foss/bencode-tools/bencode-tools-2011-03-15.tar.bz2"
+CHECKSUM_SHA256="b97e37a82593d38c44448c6f5571f14e47edbb6545391fab9ffb631bbdd2abec"
+SOURCE_DIR="bencode-tools-2011-03-15"
+PATCHES="bencode_tools-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	bencode = $portVersion
+	cmd:bencat = $portVersion
+	lib:libbencodetools = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libbencodetools
+	"
+
+PROVIDES_devel="
+	bencode_devel = $portVersion
+	devel:libbencodetools = $portVersion
+	"
+REQUIRES_devel="
+	bencode == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $developDir $binDir $libDir
+	cp -R include/* $developDir
+
+	make install
```

Comment:
$developDir is wrong above; it should be $includeDir. Also, isn't an extraneous "include" directory created because of this make install now?